packages/salesforcedx-vscode-lwc/test/vscode-integration/testSupport/commands/forceLwcTestWatchAction.test.ts

```diff
@@ -49,6 +50,23 @@ describe('Force LWC Test Watch Action', () => {
     );
   });
 
+  it('Should stop watching all tests', async () => {
```

Comment:
Can we add a negative test as well - to verify this does no harm when no tests are being watched?